windows_app_sdk/simple_window/src/main.rs

```diff
@@ -24,13 +24,13 @@ use windows::{implement, IInspectable, Interface};
 
 #[implement(extend Microsoft::UI::Xaml::Application, override OnLaunched)]
 struct App {
-    _window: Option<Window>,
+    window: Option<Window>,
```

Comment:
Underscore prefix here was to squelch warnings about an unread field. We're keeping the window alive.

Reply:
Yes you're right. I missed that the field was not being read. Sorry for the noise!